grantnav/frontend/templates/search_widgets.html

```diff
@@ -88,8 +88,8 @@ <h2>Settings:</h2>
                 </div>
                 {% include 'components/dropdown-filter.html' with title="Default Sort Order:" required=True options=dropdownFilterOptions %}
                 <div class="dropdown-filter flex-centre">
-                  <label class="dropdown-filter__label" for="select-views">Recency:</label>
-                  <select class="dropdown-filter__select" name="views" id="select-recency" onchange="updateRecency()">
+                  <label class="dropdown-filter__label" for="select-recency">Recency:</label>
+                  <select class="dropdown-filter__select" name="recency" id="select-recency" onchange="updateRecency()">
```

Comment:
This looks like it might change a form key ...but I think this just uses the `select`s value in the JS (updateRecency()) and isn't an actual submitted form so should be all fine. Shout if that isn't your understanding!